### PR TITLE
Create umeng.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by J0WI** [comment]
WFM:
umeng.com
beta.umeng.com
bugbeta.umeng.com
i.umeng.com
wsq.umeng.com
api.wsq.umeng.com

mixed content:
salon.umeng.com

Refused:
huodong.umeng.com

Timeout:
blog.umeng.com

Mismatch:
opentest.rest.api.wsq.umeng.com

Seems obsolete (returns 500)
static.umeng.com

File: src/chrome/content/rules/umeng.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	<target host="mobile.umeng.com" />
+	<target host="s.umeng.com" />
+	<target host="salon.umeng.com" />
+	<target host="static.umeng.com" />
+	<target host="wsq.umeng.com" />
+	<target host="api.wsq.umeng.com" />
+	<target host="www.umeng.com" />
+	
+	<!--mixed content:-->
+	<exclusion pattern="^http://salon.umeng.com/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/umeng.xml]: There are various deep links with this problem, you should find a better regex and test urls for it or exclude it completely.

Comment (renyouguo) [inline src/chrome/content/rules/umeng.xml]: Sorry for error example?

Comment (J0WI) [inline src/chrome/content/rules/umeng.xml]: Yes, but please use a ruleset name that makes sense.

Comment (J0WI) [inline src/chrome/content/rules/umeng.xml]: Please also provide some test urls for the exclusion regex.

**Comment by renyouguo** [comment]
@J0WI Done.

**Pull request merged**